utils.py

```diff
@@ -72,6 +73,32 @@ def mutations_data(self):
             }
         )
 
+    def edges_data(self):
+        ts = self.ts
+        edges_parent = ts.edges_parent
+        edges_child = ts.edges_child
+
+        df = pd.DataFrame(
+            {
+                "left": ts.edges_left,
+                "right": ts.edges_right,
+                "parent": edges_parent,
+                "child": edges_child,
+                "parent_time": ts.tables.nodes.time[edges_parent],
```

Comment:
It's better to use ``ts.nodes_time`` rather than ``ts.tables.nodes.time`` because ``ts.tables`` incurs a full copy of the table data each time you access it (a long-standing problem, https://github.com/tskit-dev/tskit/issues/760) whereas ``ts.node_time`` is a fast view on the underlying memory